app/uk/gov/hmrc/config/ConfigurationOps.scala

```diff
@@ -52,4 +34,5 @@ trait BaseConfig {
                (implicit find: String => Option[A]): A =
       find(key).getOrElse(default)
```

Comment:
find() might take configuration as another parameter so that we don't have to define an implicit configuration in PersonalDetailsValidationMongoRepositoryConfig